docs/reference/commandline/restart.md

```diff
@@ -25,6 +25,10 @@ Options:
   -t, --time int   Seconds to wait for stop before killing the container (default 10)
 ```
 
+## Description
+
+The main process inside the container will sleep before stopping the container.
```

Comment:
I don't think this description is correct, as the container's process won't "sleep". The `-t` / `--time` option sets a timeout before which the container's process should exit (after receiving a `SIGTERM`), if the daemon detects that the process did not exit after `-t` seconds, another signal (`SIGKILL`) is sent, to forcibly kill the container's process.